Orange/preprocess/reprable.py

```diff
@@ -0,0 +1,39 @@
+class Reprable:
```

Comment:
As said, I would move this out of _preprocess_ as its applications are much wider.